lib/mayaUsd/ufe/UsdTransform3dMayaXformStack.cpp

```diff
@@ -101,7 +102,7 @@ namespace ufe {
 
 namespace {
```

Comment:
@ppt-adsk  Unrelated to this PR but I believe having an anonymous namespace inside MAYAUSD_NS_DEF::ufe namespace is very confusing. Considering there is already an anonymous namespace starting at line 40 and ending at line 98.